PerfettoCds/PerfettoCds.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Google.Protobuf" Version="3.17.3">
+      <GeneratePathProperty>true</GeneratePathProperty>
+    </PackageReference>
+    <PackageReference Include="Microsoft.Performance.SDK" Version="0.109.11-preview-g61bc0d83f6" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="trace_processor_shell.exe">
```

Comment:
We probably shouldn't / can't check this in... ? If so, need lots of copyright and attribution at a minimum. Move to a NuGet pkg?

Reply:
I can move to a NuGet package. Is there a specific Microsoft feed I should use or would it be a public nuget package?

Reply:
Discussed offline. It would be best if we could get this as a NuGet package. Google already has one for Protobuf. I will pursue adding NuGet publishing to the Perfetto repo and show the Perfetto team. Even if they don't want to take over the NuGet publishing, they seem open to us doing it ourselves.

Will keep these exe here for this PR.

Reply:
Still need to add attribution.

Reply:
Took .exe out of the project for now.